zilliqa/src/exec.rs

```diff
@@ -4,13 +4,14 @@ use std::{
     collections::{hash_map::Entry, BTreeMap, HashMap},
     error::Error,
     fmt::{self, Display, Formatter},
-    mem,
+    fs, mem,
```

Comment:
Sure you want to use `std::fs` and not `tokio::fs` ? Will block processing if you use blocking ops from `std`

Reply:
I need to make all the functions in the call hierarchy `async` 🥲